Description: wheel_file.zipfile is moved in recent wheel API
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/917678
Forwarded: not-needed
Last-Update: 2018-12-30

--- python-pbr-4.2.0.orig/pbr/tests/test_packaging.py
+++ python-pbr-4.2.0/pbr/tests/test_packaging.py
@@ -380,8 +380,12 @@ class TestPackagingWheels(base.BaseTestC
         # Create a directory path to unpack the wheel to
         self.extracted_wheel_dir = os.path.join(dist_dir, wheel_name)
         # Extract the wheel contents to the directory we just created
-        wheel_file.zipfile.extractall(self.extracted_wheel_dir)
-        wheel_file.zipfile.close()
+        try:
+            wheel_file.zipfile.extractall(self.extracted_wheel_dir)
+            wheel_file.zipfile.close()
+        except:
+            wheel_file.extractall(self.extracted_wheel_dir)
+            wheel_file.close()
 
     def test_data_directory_has_wsgi_scripts(self):
         # Build the path to the scripts directory
